runners/core-java/src/test/java/org/apache/beam/runners/core/metrics/MetricsContainerImplTest.java

```diff
@@ -67,6 +68,9 @@ public void testCounterDeltas() {
     c1.inc(8L);
     assertThat(container.getUpdates().counterUpdates(), contains(
         metricUpdate("name1", 13L)));
+
+    CounterCell dne = container.tryGetCounter(MetricName.named("ns", "dne"));
+    assertEquals(dne, null);
```

Comment:
Probably worth a test that tryGetCounter returns non-null when it does exist?

Reply:
Amended the unit tests.